pom.xml

```diff
@@ -24,6 +24,7 @@
     <packaging>pom</packaging>
 
     <modules>
+        <module>reporter-sqs</module>
```

Comment:
sender-sqs since this isn't a reporter impl. That way, the maven names match the others, like zipkin-sender-okhttp